kotsadm/pkg/airgap/update.go

```diff
@@ -95,12 +95,21 @@ func UpdateAppFromAirgap(a *app.App, airgapBundle multipart.File) (finalError er
 	if err != nil {
 		return errors.Wrap(err, "failed to create temp file")
 	}
+
+	if err := task.SetTaskStatus("update-download", "Copying package files...", "running"); err != nil {
+		return errors.Wrap(err, "failed to set task status")
+	}
+
 	_, err = io.Copy(tmpFile, airgapBundle)
 	if err != nil {
 		return errors.Wrap(err, "failed to copy temp airgap")
 	}
 	defer os.RemoveAll(tmpFile.Name())
 
+	if err := task.SetTaskStatus("update-download", "Extracting package archive...", "running"); err != nil {
```

Comment:
should the message here match the one from the other file? like `Extracting files...`